packages/components/package.json

```diff
@@ -23,10 +23,12 @@
     "./utilities/*": "./dist/utilities/*",
     "./translations/*": "./dist/translations/*"
   },
-  "files": ["dist"],
+  "files": [
+    "dist"
+  ],
   "scripts": {
     "build": "node scripts/build.js",
-    "test": "pnpm build && web-test-runner --group default",
+    "test": "web-test-runner --group default",
```

Comment:
I fear that this could lead to people testing on old versions if we don't do a fresh build here. What do you think?

Reply:
Good point. This is also done in Synergy in this way. When using something like `test:watch` (which we currently are not implementing), an automatic build would be also feasable I think. I would like to create a new ticket for this, maybe we can wire it into `packages/components/web-test-runner.config.js` to make sure that it also builds?

It would also be nice in this case when we use a stripped down building process (e.g. ignoring the angular and react wrappers), as they are causing more strain on the system in the first place. What do you think?

Reply:
Definitely. The latter slows the process a lot.

Reply:
Let's do a new ticket for that.

Reply:
Done at https://github.com/SickDesignSystem/synergy/issues/141